openapi-specs/schema.json

```diff
@@ -4712,6 +4749,42 @@
         }
       }
     },
+    "/nodes/batch-status": {
```

Comment:
needs the leading _, otherwise it would collide with `/nodes/{className}`

Reply:
In my testing, I've not found this to happen. Is the concern that a user names one of their classes `batch-status` and that collides? Don't class names need to be capitalized?

Reply:
> Is the concern that a user names one of their classes batch-status and that collides?

exactly

> Don't class names need to be capitalized?

aren't we usually taking them upper and lower case in urls etc??

Reply:
I just tested out making a class with name `batch-status` and it is impossible due to the `kebab-case`; Weaviate rejects the hyphen. Would you prefer to not even have this potential issue? If so, I'll change np!

Reply:
ahhh, in this case it is fine with me!